src/pypants/build_targets/python_package.py

```diff
@@ -277,6 +277,7 @@ def _generate_python_binary_wrapper_node(
                         arg="dependencies",
                         value=ast.List(elts=[ast.Str(d) for d in dependencies]),
                     ),
+                    ast.keyword(arg="zip_safe", value=ast.Str(self.config.zip_safe)),
```

Comment:
I think you want these values to be `ast.Name(id=self.config.zip_safe)` not `ast.Str`